parse/ast.go

```diff
@@ -440,7 +449,25 @@ func (ld LocalDeclaration) validate() error {
 	return nil
 }
 
-func (ReservedStatement) validate() error { return nil }
+func (rs ReservedStatement) validate() error {
+	if isZeroValue(rs.Keyword) {
```

Comment:
could be good to add validation to check that the keyword is not  "input", "local", or ."match".

Reply:
yup